Automattic/wp-calypso
# Stats: Tweak commercial tier slider setup

No longer need this call to useEffect as part of the commercial slider setup.



Related to Quality Project #5.

- pejTkB-1zl-p2
- Follow-up to: https://github.com/Automattic/wp-calypso/pull/93103
- Part of Issue: https://github.com/Automattic/red-team/issues/109

## Proposed Changes

Removes usage of `useEffect` hook that should no longer be needed. The caller fetches the necessary tier data and can therefore safely setup the wrapped value without this callback.

## Why are these changes being made?


Minor code cleanup.

## Testing Instructions



Visit any site without a commercial plan, navigate to the commercial purchase page, and confirm the slider has the correct initial default value.

## Pre-merge Checklist
